src/Raven.Server/Documents/Replication/OutgoingReplicationHandler.cs

```diff
@@ -384,6 +384,27 @@ internal void SendHeartbeat()
                         UpdateDestinationChangeVector(replicationBatchReply);
                         OnSuccessfulTwoWaysCommunication();
                     }
+                    else
+                    {
+                        var msg = $"Received error from remote replication destination. Error received: {replicationBatchReply.Exception}";
+                        if (_log.IsInfoEnabled)
+                        {
+                            _log.Info(msg);
+                        }
+
+                        using (var txw = _configurationContext.OpenWriteTransaction())
```

Comment:
I don't think that we need to do an alert here.
We already are going to have replication topology that will report the status of replication.

For that matter, we already have the live state of the connections that we can report, without needing to leave the server.

Alert is for something that is high severity and isn't auto resolving.